Sources/CardPayments/CardClient.swift

```diff
@@ -161,6 +161,7 @@ public class CardClient: NSObject {
 
 extension CardClient: ASWebAuthenticationPresentationContextProviding {
 
+    @MainActor
```

Comment:
We start the authentication session on the main thread in the start function. 
Since UIApplication is marked @MainActor, there is compile error with complete checking option since 
it checks for explicit main thread context. This seems extreme to me, as it would break many implementations for others as well.